tests/migration/deregister_python2.pm

```diff
@@ -0,0 +1,20 @@
+# SLE15 migration tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Python2 module doesn't exist on some target products, need to remove it before migration
```

Comment:
To make it more clear, how about change it to 'doesn't exist products (since SLES15SP3)' WDYT?

Reply:
updated